app/hotels/src/allHotels/AllHotels.js

```diff
@@ -129,7 +129,7 @@ export class AllHotels extends React.Component<PropsWithContext> {
 type Props = {|
   +currency: string,
   +coordinates: Coordinates | null,
-  +openSingleHotel: (searchParams: AvailableHotelSearchInput) => void,
+  +openSingleHotel: (searchParams: AvailableHotelSearchInput) => void | boolean,
```

Comment:
Why exactly is it necessary to use `void | boolean`? It seems odd to me.

Reply:
I do not see the `void` type there.

Reply:
I will remove `void`